yarn.lock

```diff
@@ -8766,6 +8766,11 @@ js-beautify@^1.6.12:
     glob "^7.1.3"
     nopt "^5.0.0"
 
+js-big-decimal@1.3.4:
```

Comment:
please, don't use `js-big-decimal` as dependency 

do copy paste source code from https://github.com/royNiladri/js-big-decimal/tree/master/src

Reply:
because `commonjs` is not good, but `js-big-decimal` works only with `commonjs`